benchmark_runner/main/main.py

```diff
@@ -207,7 +209,8 @@ def main():
     success = True
     # azure_cluster_start_stop
     if azure_cluster_stop or azure_cluster_start:
```

Comment:
Again, are these globals from somewhere?

Reply:
Yes, its env variables

Reply:
It would be less confusing to pass the operation as an argument rather than from a global.

Reply:
You right, I have fixed the code and added 1 parameter: AZURE_CLUSTER_OPERATION
Thx for your important feedback